core/src/main/java/com/hcl/ecomm/core/services/impl/WishListServiceimpl.java

```diff
@@ -0,0 +1,167 @@
+            JSONObject wishlistresponse = new JSONObject();
+            if(HttpStatus.SC_OK == statusCode){
+                BufferedReader br = new BufferedReader(new InputStreamReader((httpResponse.getEntity().getContent())));
+                String str="";
+                String output;
+                while ((output = br.readLine()) != null) {
+
+                    str +=output;
+                }
+
+                wishlistresponse.put("response",str);
+                addToWishlistResponse.put("statusCode", statusCode);
+                addToWishlistResponse.put("message", wishlistresponse);
+            }else if(HttpStatus.SC_BAD_REQUEST == statusCode){
+                BufferedReader br = new BufferedReader(new InputStreamReader((httpResponse.getEntity().getContent())));
+                String str="";
+                String output;
+                while ((output = br.readLine()) != null) {
+
+                    str +=output;
+                }
+
+                wishlistresponse.put("response",str);
+                addToWishlistResponse.put("statusCode", statusCode);
+                addToWishlistResponse.put("message", wishlistresponse);
+               
+                LOG.error("Error while add to wishlist. status code:{} and message={}",statusCode,httpResponse.getEntity().getContent().toString());
+            }else{
+                LOG.error("Error while add to wishlist. status code:{}",statusCode);
+            }
+        } catch (Exception e) {
+            LOG.error("addToWishList method caught an exception " + e.getMessage());
+        }
+
+        return addToWishlistResponse;
+    }
+
+
+
+
+
+    @Override
+    public JsonObject getWishListItems(String customerToken) {
+        String token = "";
+        String url = "";
+        String domainName = loginService.getDomainName();
+
+        token = customerToken;
+        String getwishlistPath = getWishListItemsPath();
+        url = schema + "://" + domainName + getwishlistPath;
+
+        JsonObject WishListItems = new JsonObject();
+        LOG.debug("url : " + url);
+
+        CloseableHttpClient httpClient = HttpClients.createDefault();
+        HttpGet httpGet = new HttpGet(url);
+        httpGet.setHeader("Content-Type", "application/json");
+        httpGet.setHeader("Authorization", "Bearer " +token);
+        try {
+            CloseableHttpResponse httpResponse = httpClient.execute(httpGet);
+            Integer statusCode = httpResponse.getStatusLine().getStatusCode();
+            if(statusCode == 200)
+            {
+                responseStream = EntityUtils.toString(httpResponse.getEntity());
+            }
+            else
+            {
+                responseStream = "Failed to fetch wishlist details.";
+            }
+            WishListItems = new Gson().fromJson(responseStream, JsonObject.class);
+            LOG.info( "wishlist Items Response in Json object : " + WishListItems);
```

Comment:
Remove Log.info or change it to Log.debug

Reply:
updated to Log.debug